framework/source/class/qx/lang/Type.js

```diff
@@ -182,6 +183,18 @@ qx.Bootstrap.define("qx.lang.Type",
         this.getClass(value) === "Error" ||
         value instanceof Error)
       );
-    }
+    },
+
+
+    /**
+     * Whether the value is a Promise.
+     *
+     * @param value {var} Value to check.
+     * @return {Boolean} Whether the value is a Promise.
```

Comment:
Maybe explain the type of checking that is done? "This will only check whether the argument is an object that has a `then` method, as proposed by https://promisesaplus.com/"